horace_core/sqw/@sqw/sqw.m

```diff
@@ -11,14 +11,31 @@
         main_header
         header
         detpar
-        data
+        % data now a dependent property, below
```

Comment:
This comment is unnecessary: the dependent properties are still properties, this is keeping references in code to a previous implementation

Reply:
No, this is keeping a reference in the code to a CURRENT implementation, namely the one I am trying to override.